chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
PreAuthorize is fine for broad blocking, but it won't stop an ACB_ADMIN user from editing a Surveillance that was created by a ROLE_ADMIN user